flatgeobuf.cpp

```diff
@@ -381,11 +381,7 @@ void parse_flatgeobuf(std::vector<struct serialization_state> *sst, const char *
 	long long feature_sequence_id = -1;
 	int index_size = 0;
 	if (node_size > 0) {
-		if (!quiet) {
-			fprintf(stderr, "detected indexed FlatGeobuf: assigning feature IDs by sequence\n");
```

Comment:
Fine with removing this for now; FYI the rationale was that the sequence # in a spatially indexed FGB is the key useful for accessing a single feature in a "cloud-optimized" way to enable potential workflows where Tippecanoe can hold a reference to access whole OGC Features via another fetch. If I find an opportunity to actually implement something like that then maybe it deserves to be in an option sibling to `--use-attribute-for-id=`... like `--use-fgb-seqnum-for-id`